src/lib/ts-parser.ts

```diff
@@ -132,7 +132,7 @@ export class TsParser {
         for (const fg of fileGlobs) {
             rg_glob += ` --glob ${fg}`;
         }
-        const cmd = `${rgPath} --files-with-matches ${rg_glob} --no-ignore 'publicLog2|publicLogError2' ${this.sourceDir}`;
+        const cmd = `${rgPath} --files-with-matches ${rg_glob} --no-ignore \"publicLog2|publicLogError2\" ${this.sourceDir}`;
```

Comment:
Why double quotes, 
In line 129 we have single quotes, feels weird to mix quotes, i would have expected this to work with either one.